tests/index.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../../client/src/index.jsx';
+
+
+describe('<App />', () => {
+  it('renders three <Foo /> components', () => {
+    const wrapper = shallow(<MyComponent />);
+    expect(wrapper.find(Foo)).to.have.length(3);
+  });
```

Comment:
do you need to close out the describe here?

Reply:
Yes. Sorry I will have to update this.